win32ss/user/ntuser/window.c

```diff
@@ -937,7 +937,8 @@ IntLinkWindow(
 {
     if (Wnd == WndInsertAfter)
     {
-        ERR("IntLinkWindow -- Trying to link window 0x%p to itself!!\n", Wnd);
+        ERR("%s -- Trying to link window 0x%p to itself!!\n", __FUNCTION__, Wnd);
```

Comment:
Don't do more changes than necessary.
```suggestion
        ERR("IntLinkWindow -- Trying to link window 0x%p to itself!!\n", Wnd);
```

Reply:
My main idea is that identical strings can be merged, saving some bits of memory.

Reply:
I don't have a strong opinion regarding your fight, but IF you intend to change those lines, then remove one or two exclamation marks at the end of the sentence also.